content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -6,46 +6,163 @@ type: multi-code-lang
 code_lang_weight: 50
 ---
 
+## ASM capabilities 
+
+The following ASM capabilities are supported in the Node.js library, for the specified tracer version:
+
+| ASM capability                   | Minimum NodeJS tracer version |
+| -------------------------------- | ----------------------------|
+| Threat Detection  | 3.13.1|
+| Threat Protection  | 3.19.0  |
+| Open Source Vulnerability detection |  2.23.0 for NodeJS 12+, or 3.10.0 for NodeJS 14+ |
+| Custom Code Vulnerability detection  | 2.32.0 for NodeJS 12+, or 3.19.1 for NodeJS 14+ |
+
+The minimum tracer version to get all supported ASM capabilities for Node.js is 3.19.1.
```

Comment:
3.19.1 doesn't even exist

Reply:
The release with suspicious request blocking is **3.19.0**

Reply:
updated the version number